### ISSUE TITLE
Some Chinese words doesn't render normally on Manjaro Linux

### ISSUE THREAD

**Issue by hkunzhe** [BoostIO/Boostnote#3393]
# Current behavior

Some Chinese words doesn't render normally in preview area and other area except code editor.
![image](https://user-images.githubusercontent.com/30763967/70583652-57ad4000-1bf9-11ea-8341-14b55f500794.png)

My own setting for Editor Font Family is `"Hack", "Source Han Sans CN Medium"`, and the same for Preview Font Family.

As the screenshoot shown that "修改与分类“ doesn't render normally as code editor area.

# Expected behavior

The font family in the preview area should be render as set in the preferences Preview Font Family.

# Steps to reproduce

1. Set both Editor Font Family and Preview Font Family to `"Hack", "Source Han Sans CN Medium"`.
2. Type some Chinese words, like "修改与分类“.
3. Watch out note list area and preview area.

# Environment

- Version : v0.13.0
- OS Version and name : Manjaro Linux

**Comment by hkunzhe** [comment]
It seems that the mis-rendering font family is `Sans Regular` in my Manjaro Linux，which is probably set by default.
![image](https://user-images.githubusercontent.com/30763967/70585315-d6f14280-1bfe-11ea-876c-ae00bf4760d8.png)

So how can I customize it?

**Comment by Flexo013** [comment]
The characters seem to render just fine in the screenshot provided. They are merely pushed to the next line due to lack of space.

As for the font, I can't quite tell the difference between the fonts, so I have no clue if the correct font is used. Could you maybe provide a side by side of the the character in the wrong font and in the right font?

Note: I could be mistaken, since I don't know any Chinese.

**Comment by hkunzhe** [comment]
@Flexo013 , thanks for your reply. 
![image](https://user-images.githubusercontent.com/30763967/70682060-90701680-1cd8-11ea-9f4e-376f4a7460a2.png)

The character above is "分" rendered by previous area, and the character below is "分" rendered by text editor area (it correctly render by `Source Han Sans CN Medium`). There is slightly difference as the sreenshoot shown, but I have set the same font  "Hack", "Source Han Sans CN Medium" for both Editor Font Family and Preview Font Family.

I have very little knowledge about web development. I think the reason is probably caused by the style sheet below
```
@font-face {
      font-family: 'OpenSans';
      src: url('../resources/fonts/Lato-Regular.woff2') format('woff2'), /* Modern Browsers */
      url('../resources/fonts/Lato-Regular.woff') format('woff'), /* Modern Browsers */
      url('../resources/fonts/Lato-Regular.ttf') format('truetype');
      font-style: normal;
      font-weight: normal;
      text-rendering: optimizeLegibility;
    }
```
Therefore, changing Previous Font Family in preferences doesn't take effect.

**Comment by Sshzh** [comment]
老哥，问一下怎么从编辑器里修改字体啊，我试了试
![image](https://user-images.githubusercontent.com/46432627/72312133-0abdfd00-36c2-11ea-903b-133126144f6e.png)zhi直接这样输入好像修改不了

**Comment by Flexo013** [comment]
@Sshzh Could you provide English information? That way other can more easily pick up this issue.

**Comment by Sshzh** [comment]
@Flexo013 Sorry about this, I just want to ask him a question and I'm not good at English.That's why I used Chinese.....	
I didn't find any specific description about the font settings，so I try to put the font's name in the input field.
![image](https://user-images.githubusercontent.com/46432627/72320220-35b54a80-36dc-11ea-8775-b0c8ef8fd02b.png)
However this setting does not take effect, so I want to know how can I set the font correctly. thanks!

**Comment by hkunzhe** [comment]
@Sshzh, Try to set the value to `"Consolas", "Source Han Sans CN Normal"` and it should work. Maybe you have to change "Source Han Sans CN Normal" to "Microsoft YaHei" if you are using Windows.

**Comment by Sshzh** [comment]
@hkunzhe It's still not working....

**Comment by hkunzhe** [comment]
@Sshzh Please check if you have installed `Consolas` and `Microsoft YaHei` by doing `fc-list`.

**Comment by Sshzh** [comment]
Yes I have installed them,but still not working ... I have given up ...
